src/plugins/logchange/README.md

```diff
@@ -24,6 +24,8 @@ To use it, add it during mounting:
 
 ```sh
 kdb mount logchange.dump user:/logchange dump logchange
+# And to unmount it
+kdb umount user:/logchange
```

Comment:
All tests should only write below `/tests/<testname>`, see doc/TESTING.md

```suggestion
sudo kdb umount user:/tests/logchange
```

Reply:
I encountered this behaviour in other `README` files as well. Should I try to change it in them as well?

Reply:
Yes please. `kdb mount` and `kdb umount` always needs sudo at the moment.